RileyLinkKit/UI/RileyLinkDeviceTableViewController.swift

```diff
@@ -82,14 +80,18 @@ public class RileyLinkDeviceTableViewController: UITableViewController, TextFiel
         if appeared {
             tableView.reloadData()
         }
+        
+        rssiFetchTimer = NSTimer.scheduledTimerWithTimeInterval(3, target: self, selector: #selector(updateRSSI), userInfo: nil, repeats: true)
```

Comment:
`viewWillAppear` can run multiple times before `viewDidDisappear` runs, for example when cancelling a swift-from-left-edge gesture after starting it.